### PR TITLE
Redo async changes lost on merge

### PR DESCRIPTION
**Pull request by alexvandesande**
Some of these changes were lost on the merge and reviews. I also refactored some of the loadbar code based on looking the reviews. Also added a shorter time for the button to be disabled.

### CODE REVIEW

**Comment by alexvandesande** [comment]
@marekkirejczyk it's ready for review. For some reason it seems that the greetings page checkmark logic stopped working on the last PR, now when you add a new device or make a new backup it does the greeting page looks like the creating account one.

File: universal-login-example/src/views/Sections/HeaderWaiting.js [hunk 1]
```diff
@@ -6,12 +6,8 @@ function HeaderWaiting(props) {
...
       <div className="identicon-loading loading" />
       <div>
-        <p className="user-id">
-          Creating new {props.identityName} identity...
-        </p>
-        <p className="wallet-address">
-          0x0000000000000000000000000000000000000000
-        </p>
+        <p className="user-id">{props.identityName}</p>
+        <p className="wallet-address">Creating new identity...</p>
```
Comment (marekkirejczyk) [inline universal-login-example/src/views/Sections/HeaderWaiting.js]: ✅

File: universal-login-example/src/views/Sections/CreatingNewAccount.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
-  return (<div className="row">
-    <span className="bonus-ico icon-user-plus" />
-    <div>
-      <strong>Creating a new account</strong>
-      <p>
-        You&#39;ll get 20 <em>kliks</em>
-      </p>
+  return (
+    <div className="row">
+      <span className="bonus-ico icon-user-plus loading" />
```
Comment (marekkirejczyk) [inline universal-login-example/src/views/Sections/CreatingNewAccount.js]: ✅

File: universal-login-example/src/css/components/_button.sass [hunk 3]
```diff
@@ -6,20 +6,23 @@
...
+  background: darken(red, 30%)
+  transform: translate3d(0, 3px, 0)
+  cursor: not-allowed
+  transition-delay: 0s
 
 .btn:disabled, .btn.disabled
-    background: #8c7376
-    transform: translate3d(0, 3px, 0)
+  background: #8c7376
+  transform: translate3d(0, 3px, 0)
```
Comment (marekkirejczyk) [inline universal-login-example/src/css/components/_button.sass]: ✅

**Comment by marekkirejczyk** [comment]
Manually merged in [#181](https://github.com/UniversalLogin/UniversalLoginSDK/pull/181)

**Pull request closed without merging**